### PR TITLE
New module to add per Task Department

### PR DESCRIPTION
**Pull request by dreispt**
The `project_department` module makes the Project Department available in Tasks.
This module adds an editable field to Tasks, that may not correspond to the Project's Department.

### CODE REVIEW

File: project_task_department/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    "name": "Project Task specific Department",
+    "version": "1.0",
+    "author": "Daniel Reis",
+    "license": "AGPL-3",
+    "category": "Generic Modules/Projects & Services",
+    "description": """\
+Task store the Department they correspond to.
```
Comment (pedrobaeza) [inline project_task_department/__openerp__.py]: s/store/stores
or
s/Task/Tasks

**Comment by dreispt** [comment]
Improved the module description.
This new module implements #5.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by dreispt** [comment]
Thanks Pedro! Anyone else would like to comment?

**Comment by bwrsandman** [comment]
LGTM
:+1:

**Pull request merged**